src/replays/_replayswidget.py

```diff
@@ -322,7 +347,9 @@ def updatemyTree(self):
 
                     # Parse replayinfo into data
                     if item.info.get('complete', False):
-                        t = time.localtime(item.info.get('launched_at', item.info.get('game_time', time.time())))
+                        # 'game_time' is 'launched_at' for older replays (if that fails maybe 'game_end' is there)
```

Comment:
yes, using current-date as 'last resort' for game-date-time is confusing (because it's 'fake-data').
So an extra bucket 'data missing' or 'date missing' might be right here.